### PR TITLE
Should we allow pasting RTF data to chat?

### PR DESCRIPTION
**Pull request by BesatZardosht**

### CODE REVIEW

**Comment by jakestephens** [comment]
do JSQ message bubbles properly format RTF content?

**Comment by BesatZardosht** [comment]
> do JSQ message bubbles properly format RTF content?

It does. But we are disabling the rtf for our project.

**Comment by jakestephens** [comment]
oh sorry, I though the name of the merge was an open question. merging

File: JSQMessagesViewController/Views/JSQMessagesComposerTextView.m [hunk 1]
```diff
@@ -163,6 +164,11 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
 - (void)paste:(id)sender
 {
     if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
+        if (!shouldPasteRTF) {
+            [self setText:[UIPasteboard generalPasteboard].string];
+            [self.pasteDelegate composerTextView:self didPasteWithSender:sender];
```
Comment (jakestephens) [inline JSQMessagesViewController/Views/JSQMessagesComposerTextView.m:169]: its possible to get here with a nil pasteDelegate, is that ok?

Comment (BesatZardosht) [reply JSQMessagesViewController/Views/JSQMessagesComposerTextView.m:169]: Yes. In that case it uses default methods to paste.

**Comment by BesatZardosht** [comment]
> oh sorry, I though the name of the merge was an open question. merging

No worries.. my bad

File: JSQMessagesViewController/Views/JSQMessagesComposerTextView.m [hunk 2]
```diff
@@ -163,6 +164,11 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
 - (void)paste:(id)sender
 {
     if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
+        if (!shouldPasteRTF) {
+            [self setText:[UIPasteboard generalPasteboard].string];
```
Comment (mkernahan) [inline JSQMessagesViewController/Views/JSQMessagesComposerTextView.m:168]: When it is a photo... what does the .string give us?

Comment (BesatZardosht) [reply JSQMessagesViewController/Views/JSQMessagesComposerTextView.m:168]: nil

**Pull request merged**